doc/admin-api.md

```diff
@@ -35,6 +35,8 @@
 
 *Description*：Route matches requests based on preset rules, and loads the appropriate plugin according to the matching result, then forwarding requests to target Upstream.
 
+Note: When `Admin API` is enabled, Apache APISIX will occupy the API prefixed with `/apisix/admin`. Therefore, when you design an API, please try to avoid using it.
```

Comment:
```suggestion
Note: When `Admin API` is enabled, Apache APISIX will occupy the API prefixed with `/apisix/admin`. Therefore, when you design an API, please AVOID using it.
```

Reply:
if we use `try to` here, once users use this way, what should they do?

Reply:
updated.  
The Admin API port can be customized through `port_admin` in `conf/config.yaml`.